src/query/Index.cpp

```diff
@@ -211,6 +211,25 @@ ProjectionMatch Index::findNearestProjection(const Eigen::VectorXd &location, in
   }
 }
 
+ProjectionMatch Index::findCellInterpolation(const Eigen::VectorXd &location, int n)
```

Comment:
This will turn in something like `findNearestCell`.

Reply:
We are now falling back to nearest-projection by design. So this name is very misleading.

It could also be `findCellOrProjection`, but the name has to change.